rotkehlchen/chain/evm/decoding/aave/v3/decoder.py

```diff
@@ -116,6 +117,19 @@ def _decode_incentives(self, context: 'DecoderContext') -> DecodingOutput:
             amount_raw=context.tx_log.data[32:64],
         )
 
+    def _decode_mint_from_transfer(self, context: 'DecoderContext') -> DecodingOutput:
+        for event in context.decoded_events:
+            if (
+                event.event_type == HistoryEventType.RECEIVE and
+                event.event_subtype == HistoryEventSubType.NONE and
+                event.address == ZERO_ADDRESS
+            ):
+                event.event_subtype = HistoryEventSubType.INTEREST
```

Comment:
we have it as receive wrapped. I believe we shouldn't change it... is there a reason to introduce a new type?

Reply:
yeah, this new type is to differentiate it from the regular wrapped event. This event is for the amount that is earned from the interest and help to do its accounting differently.